tests/crypto/test_crypto.c

```diff
@@ -246,6 +282,36 @@ void test_pbkdf2_hmac_sha512() {
   }
 }
 
+void test_vector_signature_validity() {
```

Comment:
the name doesn't clear to me, please express the source of vectors
```suggestion
// test vectors: https://github.com/novifinancial/ed25519-speccheck
void test_ed25519_signature_edge case() {
```